packages/core/admin/admin/src/content-manager/components/Inputs/index.js

```diff
@@ -10,7 +10,7 @@ import { useIntl } from 'react-intl';
 
 import { useContentTypeLayout } from '../../hooks';
 import { getFieldName } from '../../utils';
-import Blocks from '../BlocksEditor';
+import { BlocksEditor } from '../BlocksEditor/BlocksEditor';
```

Comment:
We can import from BlocksEditor/index

Reply:
not anymore, when using typescript there's an eslint rule preventing index files. I checked other files converted to TS and they do it the same way